ledger/participant-state/kvutils/src/main/scala/com/daml/ledger/validator/StateSerializationStrategy.scala

```diff
@@ -15,7 +16,8 @@ final class StateSerializationStrategy(keyStrategy: StateKeySerializationStrateg
   def serializeStateUpdates(
       state: Map[DamlStateKey, DamlStateValue]
   ): SortedMap[Raw.Key, Raw.Value] =
-    state.map { case (key, value) =>
-      serializeState(key, value)
-    }(breakOut)
+    implicitly[Factory[Raw.KeyValuePair, SortedMap[Raw.Key, Raw.Value]]]
+      .fromSpecific(state.view.map { case (key, value) =>
+        serializeState(key, value)
+      })
```

Comment:
I assume here it was just impossible to do something like `.to(SortedMap)`, right?

Reply:
I’ve tried that first and it works on Scala 2.13 but I couldn’t get it to typecheck on 2.12 so at some point I gave up and went down this route.